src/tests/baseservices/typeequivalence/simple/Simple.cs

```diff
@@ -253,6 +254,13 @@ private static unsafe void TestTypeEquivalenceWithTypePunning()
         }
     }
 
+    [MethodImpl (MethodImplOptions.NoInlining)]
+    private static void TestLoadingValueTypeWithStaticMethod() 
+    {
+        Console.WriteLine($"{nameof(TestLoadingValueTypeWithStaticMethod)}");
+        Console.WriteLine($"-- {typeof(ValueTypeWithStaticMethod).Name}");
+    }
```

Comment:
I think we should also have a regression based test. Let's attempt to load a type that will throw and validate the exception type is expected. Create a new type called `ValueTypeWithMethod` and use that to verify it throws as expected. We can use a local static function like below, which can be pasted within this method. Remember to update the name of `TestLoadingValueTypeWithStaticMethod`.

```csharp
Assert.Throws<TypeLoadException>(() => LoadInvalidType());

[MethodImpl (MethodImplOptions.NoInlining)]
static void LoadInvalidType()
{
        Console.WriteLine($"-- {typeof(ValueTypeWithMethod).Name}");
}
```